disperser/controller/dispatcher.go

```diff
@@ -341,37 +349,58 @@ func (d *Dispatcher) HandleSignatures(
 	}
 	err := d.blobMetadataStore.PutAttestation(ctx, attestation)
 	if err != nil {
-		return fmt.Errorf("failed to put attestation for batch %s: %w", batchHeaderHash, err)
+		// this error isn't fatal: a subsequent PutAttestation attempt might succeed
+		// TODO: this used to cause the HandleSignatures method to fail entirely. Is it ok to continue trying here?
```

Comment:
reminder to resolve this TODO

Reply:
I'm not sure the answer here yet. Let's leave this conversation open until I can get to the bottom it it

Reply:
removed